sequencer.c

```diff
@@ -1200,6 +1200,7 @@ struct todo_list {
 	struct todo_item *items;
 	int nr, alloc, current;
 	int done_nr, total_nr;
+        struct stat_data stat;
```

Comment:
I don't mind fixing these things myself, but when you contribute this patch to the mailing list, you may get upset comments about using spaces for indentation. To avoid such comments, I recommend using tabs.